pallets/band/src/lib.rs

```diff
@@ -541,7 +541,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
         let decayed_fee = prev_fee * decay;
 
         let new_fee_part =
-            (prev_rate / new_rate - fixed_wrapper!(1)).abs() - doubled_deviation - min_fee;
+            (new_rate / prev_rate - fixed_wrapper!(1)).abs() - doubled_deviation - min_fee;
         let new_fee = if new_fee_part > fixed_wrapper!(0) {
             decayed_fee + new_fee_part
```

Comment:
You use `abs()` here. But what if `new_rate` is less than `prev_rate`? In this case we still increase `new_fee`. Is it correct?

Reply:
Good question, it may be a good idea to make the fee only grow with pumps, since we can't short the synthetics in SORA (at least now). But I'm not sure if dumps do not make positive influence during long high volatility periods. Anyway, sounds good, will fix this